### PR TITLE
updating client application form to match new schema

### PR DESCRIPTION
**Pull request by nyaculak**
Added all the fields for the new questions

### CODE REVIEW

File: app/server/models/User.js [hunk 1]
```diff
@@ -114,7 +109,7 @@ var profile = {
   whyBitcamp: {
     type: String,
     min: 0,
-    max: 2500
```
Comment (jsjqian) [inline app/server/models/User.js]: Why are we switching back to 250?

**Comment by jsjqian** [comment]
Looks good to me. Good work, Nick! I only have one comment about the why Bitcamp section. I see the TODO; hopefully we come back to it at some point. @wyapa take a look and then merge if you don't see any problems either.

**Comment by wyapa** [comment]
Looks good

File: app/server/models/User.js (hunk 1 above)
Comment (nyaculak) [reply app/server/models/User.js]: This is only supposed to be a 250 character response, as outlined in the google doc. The other response is supposed to be a 100 word response, so it's set to be 1000 characters.

**Pull request merged**